src/nvim/tui/tui.c

```diff
@@ -805,6 +841,16 @@ static void tui_set_icon(UI *ui, String icon)
 // to make a copy for the tui thread
 static void tui_event(UI *ui, char *name, Array args, bool *args_consumed)
```

Comment:
this shouldn't be used. In fact ui_event impl should be just deleted from ui_bridge and tui. (the infrastructure checks for funptr being NULL, so this is safe). Instead, mark events as not REMOTE_ONLY to have them acessible from TUI.